2020q4/k8s-bhyve.md

```diff
@@ -0,0 +1,16 @@
+## K8S-bhyve ##
+
+Link:	 [K8S-bhyve](https://k8s-bhyve.convectix.com)
+Link:	 [K8S-bhyve](https://github.com/k8s-bhyve)
+Link:	 [Kubernetes](https://kubernetes.io/)
+
+Contact: Kirill Ponomarev <krion@FreeBSD.org>
+Contact: Oleg Ginzburg <olevole@olevole.ru>
+
+K8S-bhyve is opensource project concentrating primarily to deploy and use
+kubernetes on FreeBSD/bhyve in more agile and more comfortable. We are going
+to provide distributed multi-DC environment or just stand-alone clusters with
+native PV/PVC support.
```

Comment:
in *a* more agile and more comfortable *manner* perhaps?

Reply:
yeah, sorry for my rusty English